Compiler error due to not found message headers

Hey,
I tried using the TransformBroadcaster in my own program (c++). 
After adding the include

``` c++
#include <tf2_ros/transform_broadcaster.h>
```

I got the following errors from the transform_broadcast.h file:
First it couldnt find the #include "geometry_msgs/msg/transform_stamped.h"
I solved this by changing the include to:

``` c++
#include "geometry_msgs/msg/transform_stamped.hpp"
```

Second it couldnt find the tf2_msgs::msg::TFMessage
I fixed this by adding the include

``` c++
#include "tf2_msgs/msg/tf_message.hpp"
```

I'm using the version that comes with ros2 alpha 3. 

How: I'm having a project where I include the file:

``` c++
#include <tf2_ros/transform_broadcaster.h>
```

Error:
I get this error and some more errors all complaining about that it can't find tf2_msgs

```
~/workspace/ros_ws/install/include/tf2_ros/transform_broadcaster.h:72:32: error: ‘tf2_msgs’ was not declared in this scope
   rclcpp::publisher::Publisher<tf2_msgs::msg::TFMessage>::SharedPtr publisher_;
```

Solution:

In the file: ~/workspace/ros_ws/install/include/tf2_ros/transform_broadcaster.h add the following line:

``` c++
#include "tf2_msgs/msg/tf_message.hpp"
```

What is kind of strange that I can't reproduce the error that says it can't find the geometry messages. I think I did something wrong myself there....

--- a/tf2_ros/include/tf2_ros/transform_broadcaster.h
+++ b/tf2_ros/include/tf2_ros/transform_broadcaster.h
@@ -36,7 +36,7 @@
 
 
 #include "rclcpp/rclcpp.hpp"
-#include "geometry_msgs/msg/transform_stamped.h"
+#include "geometry_msgs/msg/transform_stamped.hpp"
 namespace tf2_ros
 {
 

